samples/vsphere/sso/external_psc_sso_workflow.py

```diff
@@ -82,7 +82,8 @@ def options(self):
         self.args = self.argparser.parse_args()
 
     def setup(self):
-        self.lswsdl = self.args.lswsdl
+        if(self.args.lswsdl):
```

Comment:
Yes..even if wsdlpath is not passed as argument, os.path.abspath gets called on None object, which is throwing error. Hence I added If for the checking argument.

Squashed commits